lib/game_of_life.ex

```diff
@@ -1,18 +1,3 @@
 defmodule GameOfLife do
-  @moduledoc """
```

Comment:
What is this skeleton that is already in your initial commit? You initial commit should be a .gitignore and README. This stuff should not be merged into master. Please fix this.